testing/simapp/app.go

```diff
@@ -217,6 +222,7 @@ type SimApp struct {
 	// make IBC modules public for test purposes
 	// these modules are never directly routed to by the IBC Router
 	ICAAuthModule ibcmock.IBCModule
+	FeeMockModule ibcmock.IBCModule
```

Comment:
This is the base application, it needs to be the mock module since that's the type we can change the callbacks for as done in the `OnChanOpenInit` test. This prevents us from hard coding certain logic within the mock module itself and rather specifying the expected behaviour from the base application within the test itself